src/main/java/tech/sourced/siva/SivaReader.java

```diff
@@ -0,0 +1,48 @@
+package tech.sourced.siva;
+
+import java.io.*;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+
+public class SivaReader {
+
+    private final RandomAccessFile sivaFile;
+    private final FileChannel channel;
+
+    public SivaReader(File sivaFile) throws FileNotFoundException {
```

Comment:
I see what you mean - on one side this reasoning make sense: why shall SivaReader handle errors it's not responsible for creating?

On another side - this will unnecessary complicate the siva-java client code: it needs to have one extra catch clause, where he can not do anything meaningful (re-try, re-start, etc), except for just skipping it  :/